.gitmodules

```diff
@@ -1,3 +1,5 @@
 [submodule "packages/core/temp/lib/forge-std"]
 	path = packages/core/temp/lib/forge-std
 	url = https://github.com/foundry-rs/forge-std
+[submodule "lib/forge-std"]
```

Comment:
I ran `yarn clean` and `yarn qbuild` and this happened. Should it be here?

Reply:
@chrismaree is this expected? Or should we drop this from the diff?

Reply:
I think it should? im not sure why it was not pulled in previously but we use this via submodules.